frontend/src/screens/wallet/index.tsx

```diff
@@ -69,24 +70,7 @@ function Wallet() {
         !!channels?.length &&
         balances.lightning.totalReceivable <
           balances.lightning.totalSpendable * 0.1 && (
-          <Alert>
-            <AlertTriangleIcon className="h-4 w-4" />
-            <AlertTitle>Low receiving capacity</AlertTitle>
-            <AlertDescription>
-              You likely won't be able to receive payments until you{" "}
-              <Link className="underline" to="/wallet/send">
-                spend
-              </Link>
-              ,{" "}
-              <Link className="underline" to="/wallet/swap?type=out">
-                swap out funds
-              </Link>
-              , or{" "}
-              <Link className="underline" to="/channels/incoming">
-                increase your receiving capacity.
-              </Link>
-            </AlertDescription>
-          </Alert>
+          <LowReceivingCapacityAlert />
```

Comment:
below this, the "open your first channel" alert now has weird spacing - Link seems to no longer be inline:
<img width="896" height="254" alt="image" src="https://github.com/user-attachments/assets/7853cf8d-77ab-4731-8361-c1284a7af46b" />

Reply:
Also buttons seem to have too much spacing 
<img width="602" height="296" alt="image" src="https://github.com/user-attachments/assets/0cdef0ef-f2f4-4dac-bcd7-b8f5bf364ee0" />

Reply:
Yeah, that's cool!